src/server/page_server.cpp

```diff
@@ -138,12 +138,14 @@ page_server::connection_handler::receive_data_page_fetch (tran_server_conn_t::se
 }
 
 void
-page_server::connection_handler::receive_log_boot_info_fetch (tran_server_conn_t::sequenced_payload &)
+page_server::connection_handler::receive_log_boot_info_fetch (tran_server_conn_t::sequenced_payload &a_sp)
 {
   // empty request message
 
-  auto callback_func =
-	  std::bind (&connection_handler::on_log_boot_info_result, this, std::placeholders::_1);
+  auto callback_func = [this, sp = a_sp] (std::string &&message) mutable
```

Comment:
My intention was to use move semantics, but with all my struggle I could not find a consistent way to move the payload into the the bounded function and then move it again when the function is invoked. After half a day of trying out various suggestions, I gave up.

I think this may get another shot with LETS-329, when maybe async page fetcher could be rewritten in a way that this becomes possible (e.g. move the payload into the task).